ops/solana/solana.go

```diff
@@ -48,6 +53,11 @@ func New(ctx *pulumi.Context) (Deployer, error) {
 	cwd, _ := os.Getwd()
 	os.Chdir(filepath.Join(cwd, "../gauntlet"))
 
+	fmt.Println("Installing dependencies")
```

Comment:
Let's add a Makefile in the near future that will script some of these steps for us, and also provide a close to unified interface across relay integration repos.